src/events/attributes.rs

```diff
@@ -32,6 +32,25 @@ pub struct Attribute<'a> {
 }
 
 impl<'a> Attribute<'a> {
+    /// Creates new attribute from raw bytes.
+    /// Does not apply any transformation to either key or value.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use pretty_assertions::assert_eq;
+    /// use quick_xml::events::attributes::Attribute;
+    ///
+    /// let features = Attribute::new("features".as_bytes(), "Bells &amp; whistles".as_bytes());
```

Comment:
I think, this looks better:
```suggestion
    /// let features = Attribute::new(b"features", b"Bells &amp; whistles");
```

Reply:
The type of this is `[u8; <len>]` rather than `&[u8]`, so you have to do a bit extra, and at that point it's not cleaner anymore.

Reply:
This is only when you apply that suggestion to the tuple, because `(b"...", b"...")` will have a type `([u8; <len1>], [u8; <len2>])`. Using byte literals with `from_*` methods is compiled, I checked